xml/deployment_firstboot.xml

```diff
@@ -0,0 +1,870 @@
+   <procedure>
+    <step>
+     <para>
+      Open the <filename>/etc/YaST2/firstboot.xml</filename> configuration
+      file.
+     </para>
+    </step>
+    <step>
+     <para>
+      Change the value for the <literal>enabled</literal> element from
+      <literal>true</literal> to <filename>false</filename> to disable the
+      module or from <literal>false</literal> to <literal>true</literal> to
+      enable it again.
+     </para>
+<screen>
+&lt;module>
+    &lt;label>Time and Date&lt;/label>
+    &lt;enabled config:type="boolean">true&lt;/enabled>
+    &lt;name>firstboot_timezone&lt;/name>
+&lt;/module>
+</screen>
+    </step>
+    <step>
+     <para>
+      Apply your changes and close the configuration file.
+     </para>
+    </step>
+   </procedure>
+   <para>
+    To add a custom made module to the workflow, proceed as follows:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Create your own &yast; module and store the module file
+      <filename><replaceable>module_name</replaceable>.rb</filename> in
+      <filename>/usr/share/YaST2/clients</filename>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Open the <filename>/etc/YaST2/firstboot.xml</filename> configuration
+      file.
+     </para>
+    </step>
+    <step>
+     <para>
+      Determine at which point in the workflow your new module should be
+      run. In doing so, make sure that possible dependencies to other steps
+      in the workflow are taken into account and resolved.
+     </para>
+    </step>
+    <step>
+     <para>
+      Create a new <literal>module</literal> element inside the
+      <literal>modules</literal> container and add the appropriate
+      sub-elements:
+     </para>
+<screen>&lt;modules config:type="list">
+     &hellip;
+     &lt;module>
+        &lt;label><replaceable>my_module</replaceable>&lt;/label>
+        &lt;enabled config:type="boolean">true&lt;/enabled>
+        &lt;name><replaceable>filename_my_module</replaceable>&lt;/name>
+     &lt;/module>
+&lt;/modules>
+</screen>
+     <substeps>
+      <step>
+       <para>
+        Enter the label to be displayed on your module in the
+        <literal>label</literal> element.
+       </para>
+      </step>
+      <step>
+       <para>
+        Make sure that <literal>enabled</literal> is set to
+        <literal>true</literal> to have your module included in the
+        workflow.
+       </para>
+      </step>
+      <step>
+       <para>
+        Enter the file name of your module in the <literal>name</literal>
+        element. Omit the full path and the <filename>.rb</filename>
+        suffix.
+       </para>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      Apply your settings and close the configuration file.
+     </para>
+    </step>
+   </procedure>
+   <tip>
+    <title>Finding Connected Network Interface For Auto-Configuration</title>
+    <para>
+     If the target hardware may feature more than one network interface add
```

Comment:
```suggestion
     If the target hardware may feature more than one network interface, add
```

Reply:
Additional suggestion for simplicity: "may feature" -> "may have"